ts/modules/accessibility/components/SeriesComponent/SeriesDescriber.ts

```diff
@@ -579,15 +579,17 @@ function describeSeriesElement(
  * @param {Highcharts.Series} series The series to add info on.
  */
 function describeSeries(series: Highcharts.AccessibilitySeries): void {
-    var chart = series.chart,
+    const chart = series.chart,
         firstPointEl = getSeriesFirstPointElement(series),
-        seriesEl = getSeriesA11yElement(series);
+        seriesEl = getSeriesA11yElement(series),
+        is3d = chart.options.chart?.options3d?.enabled;
```

Comment:
Wouldn't be better to use `chart.is3d && chart.is3d()`?

Reply:
Oh cool, was not aware of this function. Will change